src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -135,6 +135,7 @@
     <InternalsVisibleToRemoteLS Include="Microsoft.VisualStudio.LanguageServices.Remote.CSharp" />
     <InternalsVisibleToFSharp Include="FSharp.Editor" />
     <InternalsVisibleToFSharp Include="FSharp.LanguageService" />
+    <InternalsVisibleToRazor Include="Microsoft.VisualStudio.LanguageServices.Razor"/>
```

Comment:
This itemgroup is already defined because we have IVT access to a few shim assemblies that build from the repo already.